imports/ui/layouts/app-body.js

```diff
@@ -121,7 +120,7 @@ Template.App_body.events({
     Meteor.logout();
 
     // if we are on a private list, we'll need to go to a public one
-    if (ActiveRoute.name('Lists.show')) {
+    if (FlowRouter.current().route.name === 'Lists.show') {
```

Comment:
```js
import { FlowRouter , RouterHelpers } from 'meteor/ostrio:flow-router-extra';

RouterHelpers.name('Lists.show');
```

See [docs for more details](https://github.com/VeliovGroup/flow-router/blob/master/docs/helpers/RouterHelpers.md)